compose/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+version: '2'
+services:
+  # proftpd container
+  proftp:
+    image: quay.io/galaxy/proftpd:compose
+    container_name: galaxy-proftp
+    volumes:
+        - ftp_config:/etc/proftpd/
+        - ftp_data:/export/ftp
+    expose:
+        - 21
+    ports:
+        - "8021:21"
+    # proftpd config files from Galaxy are needed
+    depends_on:
+        - galaxy
+    links:
+        - galaxy:galaxy.local
+    restart: always
+    hostname: galaxy-proftp
+
+
+  # Main Galaxy container
+  galaxy:
+    image: quay.io/bgruening/galaxy:compose
+    environment:
+        #    Insert here your environment variables to change Galaxy's behaviour.
+        #    For example to personalise Galaxy by changing the branding with
+        #    - GALAXY_CONFIG_BRAND=FOO
+        - NONUSE=proftp
+        - GALAXY_CONFIG_FTP_UPLOAD_DIR=/export/ftp
+        - GALAXY_CONFIG_ALLOW_USER_DATASET_PURGE=True
+        - GALAXY_CONFIG_ALLOW_LIBRARY_PATH_PASTE=True
+        - GALAXY_CONFIG_ENABLE_USER_DELETION=True
+        - GALAXY_CONFIG_ENABLE_BETA_WORKFLOW_MODULES=True
+    ports:
+        - "8080:80" # nginx
+        - "9002:9002" # supervisor web-interface - not super useful in compose mode
+    expose:
+        # The proftp container needs Galaxy's postgres to check authentification
+        - 5432
```

Comment:
@bgruening 
I don't think you need to (or should) expose this, cause you are using the same docker network you can just contact the containers using their hostname. In return 0.0.0.0/0 should be safe in postgresql.conf